synfig-studio/src/gui/widgets/widget_timeslider.cpp

```diff
@@ -303,23 +309,52 @@ Widget_Timeslider::on_draw(const Cairo::RefPtr<Cairo::Context> &cr)
 				double x1 = time_plot_data->get_double_pixel_t_coord(bounds[i][1]);
 				double w = x1 - x0;
 
+				double boundary_dimension=8;
+				double background_adjust;
+
+				if(i==0)
+					background_adjust=0;
+				else
+					background_adjust=8;
```

Comment:
IMO it's clearer if it was in a single (and simple) line:
```c++
double background_adjust = (i == 0) ? 0 : 8;
```
It's a safer habit too: always initialize a (primitive) variable.

I guess this `8` should actually be `boundary_dimension`.

Reply:
hmm, yes the ternary operator in my opinion also here makes things nice and compact, but I don't usually opt for it as some/many people deem it "bad for readability".